.vscode/launch.json

```diff
@@ -290,13 +290,29 @@
-              "NEW_DEPS_VERSION": "8.0.0-preview.23465.2",
+              "NEW_DEPS_URLS": "https://download.visualstudio.microsoft.com/download/pr/e0c516ea-aa15-448e-8fee-a2f27785ee9e/1f7ca88955dd8ee28a016c7b699de1a8/razorlanguageserver-linux-arm64-7.0.0-preview.23472.3.zip,https://download.visualstudio.microsoft.com/download/pr/e0c516ea-aa15-448e-8fee-a2f27785ee9e/e2b17d54dfea4ce7a1ebc3e20f93e006/razorlanguageserver-linux-musl-arm64-7.0.0-preview.23472.3.zip,https://download.visualstudio.microsoft.com/download/pr/e0c516ea-aa15-448e-8fee-a2f27785ee9e/1a9c9111b37d3254d1c6edbdb62fa382/razorlanguageserver-linux-musl-x64-7.0.0-preview.23472.3.zip,https://download.visualstudio.microsoft.com/download/pr/e0c516ea-aa15-448e-8fee-a2f27785ee9e/8df1db725708594806369c7e8a547a02/razorlanguageserver-linux-x64-7.0.0-preview.23472.3.zip,https://download.visualstudio.microsoft.com/download/pr/e0c516ea-aa15-448e-8fee-a2f27785ee9e/f6cfc265a4e01a4bc6e493abdeab3e4c/razorlanguageserver-osx-arm64-7.0.0-preview.23472.3.zip,https://download.visualstudio.microsoft.com/download/pr/e0c516ea-aa15-448e-8fee-a2f27785ee9e/34134fce2a63158bcf5b7e8c3813289a/razorlanguageserver-osx-x64-7.0.0-preview.23472.3.zip,https://download.visualstudio.microsoft.com/download/pr/e0c516ea-aa15-448e-8fee-a2f27785ee9e/4534001b203012f51e0bd897c16dd414/razorlanguageserver-win-arm64-7.0.0-preview.23472.3.zip,https://download.visualstudio.microsoft.com/download/pr/e0c516ea-aa15-448e-8fee-a2f27785ee9e/0156660d3533efef9a0563a6c95686d9/razorlanguageserver-win-x64-7.0.0-preview.23472.3.zip,https://download.visualstudio.microsoft.com/download/pr/e0c516ea-aa15-448e-8fee-a2f27785ee9e/43aae1d74406b86393bd98af202e4b73/razorlanguageserver-win-x86-7.0.0-preview.23472.3.zip",
+              "NEW_DEPS_VERSION": "7.0.0-preview.23472.3",
```

Comment:
It might look weird that we're going from 8.0.0* -> 7.0.0* here, but the latter version is actually newer. In between these two changes, Razor reverted a change to switch version prefixes to 8.* due to some ngen issues in VS